docs/tex/api/reference.tex

```diff
@@ -0,0 +1,93 @@
+\title{Reference}
+
+{{navbar}}
+
+\subsubsection{Models}
+
+{{sphinx
+
+* :class:`edward.models.RandomVariable`
+  
+  * :class:`edward.models.Bernoulli`
```

Comment:
Sphinx has something build in (intersphinx), if TF has a sphinx based docs. I will look into it. Otherwise, the URL format should not be too difficult to reverse engineer.